README.md

```diff
@@ -1018,6 +1018,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Kunal Ranjan](https://github.com/kunalranjan19)
   - [Kunal Sangtiani](https://github.com/Kunals131)
   - [Kuwar Kapur](https://github.com/kuwarkapur)
+  - [Khalid Hasan](https://github.com/khalidx3)
```

Comment:
Name should be in alphabetical order. Please add your name in line **984**, after Khalid Bayan. 
If you need any help with that, let us know.

Reply:
I have committed the changes do I need to raise the pr again? if yes then how can I do that?